doc: Monthly product updates - September 2023

pilvikala opened the pull request:
#### Summary

monthly product update for September 2023

jsonpr commented on website/pages/blog/product-updates-8.mdx:
```diff
@@ -0,0 +1,58 @@
+
+## SDKs!
+
+ICYMI, we published [Python](https://www.cloudquery.io/blog/announcing-cloudquery-python-sdk) and [JavaScript](https://www.cloudquery.io/blog/announcing-cloudquery-javascript-sdk) SDKs last month and this month, we followed with Java!
```
I'd suggest we take out the abbreviations such as ICYMI and TLDR in case readers don't understand.

jsonpr commented on website/pages/blog/product-updates-8.mdx:
```diff
@@ -0,0 +1,58 @@
+
+- Open source: A new SDK, new plugins, performance improvements
+- Premium (closed source): New policies
+
```
I'd move the AWS Partner announcement here as a highlight.

jsonpr commented on website/pages/blog/product-updates-8.mdx:
```diff
@@ -0,0 +1,58 @@
+
+- [Bitbucket](https://www.cloudquery.io/docs/plugins/sources/bitbucket/overview) - First plugin utilizing our Java SDK!
+- [Notion](https://github.com/cloudquery/cloudquery/tree/main/plugins/source/notion) - Loads databases, pages, and users from Notion to any database, data warehouse or data lake supported by CloudQuery, such as PostgreSQL, BigQuery, Athena, and many more.
+- You can now configure the API version on the [Shopify](https://www.cloudquery.io/docs/plugins/sources/shopify/overview) plugin.
```
This is a nit, does configuring the API version fall under "new source plugins"?

jsonpr commented on website/pages/blog/product-updates-8.mdx:
```diff
@@ -0,0 +1,58 @@
+
+### AWS, GCP, Azure Highlights
+
+- We launched a closed beta for event-based sync with the AWS plugin. This helps you keeping your data fresh without running full syncs. [Sign up for beta](https://cloudquery.typeform.com/to/Dvdn6P2u) or [read the full announcement](https://www.cloudquery.io/blog/announcing-cloudquery-aws-event-based-sync-beta) first.
```
Grammar update
```suggestion
- We launched a closed beta for event-based sync with the AWS plugin. This helps with keeping your data fresh without running full syncs. [Sign up for beta](https://cloudquery.typeform.com/to/Dvdn6P2u) or [read the full announcement](https://www.cloudquery.io/blog/announcing-cloudquery-aws-event-based-sync-beta) first.
```

jsonpr commented on website/pages/blog/product-updates-8.mdx:
```diff
@@ -0,0 +1,58 @@
+
+- We launched a closed beta for event-based sync with the AWS plugin. This helps you keeping your data fresh without running full syncs. [Sign up for beta](https://cloudquery.typeform.com/to/Dvdn6P2u) or [read the full announcement](https://www.cloudquery.io/blog/announcing-cloudquery-aws-event-based-sync-beta) first.
+
+- AWS has got a new scheduler strategy that reduces the number of concurrent calls made to the account contact APIs in syncs with multiple tables. Read more about this in our documentation on [performance tuning](https://www.cloudquery.io/docs/advanced-topics/performance-tuning#use-a-different-scheduler).
```
Grammar edit to remove "has got"
```suggestion
- The AWS plugin has a new scheduler strategy that reduces the number of concurrent calls made to the account contact APIs in syncs with multiple tables. Read more about this in our documentation on [performance tuning](https://www.cloudquery.io/docs/advanced-topics/performance-tuning#use-a-different-scheduler).
```

jsonpr replied:
I would also make this a more general statement instead of focusing on the "account contact APIs"

jsonpr commented on website/pages/blog/product-updates-8.mdx:
```diff
@@ -0,0 +1,58 @@
+
+## Policies
+
+- We released [AWS Foundational Security Best Practices for Snowflake!](https://www.cloudquery.io/blog/announcing-aws-fsbp-policies-snowflake)
```
Would be good to highlight numbers on this post.  This release has more than  200 controls with coverage for more than 40 AWS services.

yevgenypats reviewed (approved):
Nice!

pilvikala commented:
@jsonpr I think I have now addressed all your comments.

Note: waiting for the permission from the author of the Notion plugin to mention their name in the newsletter.
